turbopack/crates/turbopack-ecmascript-runtime/js/src/browser/runtime/base/dev-base.ts

```diff
@@ -126,6 +126,20 @@ const getOrInstantiateModuleFromParent: GetOrInstantiateModuleFromParent<
   return instantiateModule(id, SourceType.Parent, sourceModule.id)
 }
 
+function DevContext(
```

Comment:
why not use `class` syntax?

Reply:
i suppose i see why explicit prototype is a bit simpler to integrate given our manually obfuscated names.

plus class would require a mixin approach 🤮  or a subclass to shared parts of the implementation across the different runtime variants,

is that it or am i missing something else?